docs/_articles/en/getting-started/install.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Install Salesforce Scanner CLI Plugin
+title: Install Salesforce Code Analyzer Plugin
 lang: en
 ---
 
 ## Install the plug-in
 
-Like all the Salesforce CLI plug-ins, you install the Scanner CLI plug-in with one simple step. NOTE: Be sure you've completed the [prerequisites](./en/getting-started/prerequisites/) before you install this plug-in.
+Like all the Salesforce CLI plug-ins, you install the Analyzer CLI plug-in with one simple step. NOTE: Be sure you've completed the [prerequisites](./en/getting-started/prerequisites/) before you install this plug-in.
```

Comment:
Should we call it the "Code Analyzer" plugin instead of simply "Analyzer"? We were previously dropping only the "Salesforce" prefix. Calling it "Code Analyzer" would be consistent.